Gemfile

```diff
@@ -71,6 +71,7 @@ gem "rswag-api"
 gem "rswag-ui"
 gem "ruby-progressbar", require: false
 gem "rubyzip"
+gem "zip_tricks"
```

Comment:
Moreover, the `zip_tricks` gem is written in the [Hippocratic License](https://firstdonoharm.dev/version/1/2/license/) which is not a valid Open Source license because it has a non-enforceable and nebulous "do no harm" clause.